mmv1/third_party/terraform/fwtransport/framework_config.go.erb

```diff
@@ -31,30 +31,25 @@ import (
 )
 
 type FrameworkProviderConfig struct {
-    BillingProject types.String
-    Client *http.Client
-    Context context.Context
-    gRPCLoggingOptions []option.ClientOption
-    PollInterval time.Duration
-    Project types.String
-    Region types.String
-    Zone types.String
-    RequestBatcherIam *transport_tpg.RequestBatcher
-    RequestBatcherServiceUsage *transport_tpg.RequestBatcher
-    Scopes []string
-    TokenSource oauth2.TokenSource
-    UserAgent string
-    UserProjectOverride types.Bool
-
-    // paths for client setup
-    <% products.each do |product| -%>
-    <%= product[:definitions].name -%>BasePath string
-    <% end -%>
-}
-
-var defaultClientScopes = []string{
-	"https://www.googleapis.com/auth/cloud-platform",
-	"https://www.googleapis.com/auth/userinfo.email",
```

Comment:
This `defaultClientScopes` was removed because it's a duplicate of `transport_tpg.DefaultClientScopes`